docs/database/database-spans.md

```diff
@@ -67,30 +67,24 @@ Some database systems may allow a connection to switch to a different `db.user`,
 | `db.system` | string | An identifier for the database management system (DBMS) product being used. See below for a list of well-known identifiers. | `other_sql` | Required |
 | `db.connection_string` | string | The connection string used to connect to the database. It is recommended to remove embedded credentials. | `Server=(localdb)\v11.0;Integrated Security=true;` | Recommended |
 | `db.user` | string | Username for accessing the database. | `readonly_user`; `reporting_user` | Recommended |
+| [`network.server.address`](../general/attributes.md) | string | Server address of the (physical) network connection - domain name if available without reverse DNS lookup, otherwise IP address or Unix domain socket name. [1] | `example.com`; `10.1.2.80`; `/tmp/my.sock`; `proxy.example.com` | Recommended: If different than `server.address`. |
+| [`network.server.ip`](../general/attributes.md) | string | Server IP address of the (physical) network connection. | `10.1.2.80` | Recommended: If different than `network.server.address`. |
```

Comment:
I kinda like `network.peer.*` `network.local.*` more since they would be useful for symmetrical network peers (e.g. gossip protocols),

Reply:
do you prefer `network.local.*`/`network.peer.*` over `network.local.*`/`network.remote.*`?

Reply:
probably `network.peer.*`, but no strong opinion. See my comment below - I don't think we need to block HTTP semconv on this.